content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Isito mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 3
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to install an Istio control plane
+[multicluster deployment](/docs/ops/deployment/deployment-models/#multiple-clusters) and connect your first virtual machine.
+
+## Prerequisites
+
+- One or more Kubernetes clusters with versions: {{< supported_kubernetes_versions >}}.
+
+- Authority to [deploy the Istio control plane](/docs/setup/install/istioctl/).
+
+- Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, or alternatively, every pod in the mesh via L3 networking.
+
+- Services in the cluster must be accessible through the ingress gateway.
+
+## Installing the Istio control plane
+
+1. Generate intermediate CA certificates for each cluster's CA from your
```

Comment:
For accuracy, this should be an idea callout or some other form of notice, rather than a step.